packages/orchestration/src/util/filtering.ts

```diff
@@ -65,14 +65,16 @@ export function buildAzureContentSafetyFilter<T extends 'input' | 'output'>(
 }
 
 /**
- * Convenience function to build Llama guard filter.
+ * Convenience function to build Llama Guard 3 8B filter.
+ * @param type - Type of the filter, either 'input' or 'output'.
  * @param categories - Categories to be enabled for filtering. Provide at least one category.
- * @returns Filter config object.
- * @example "buildLlamaGuardFilter('self_harm', 'hate')"
+ * @returns Llama Guard 3 8B filter configuration.
+ * @example "buildLlamaGuard38BFilter('input', ['elections', 'hate'])"
```

Comment:
```suggestion
 * @example buildLlamaGuard38BFilter('input', ['elections', 'hate'])
```
`"` removes the code highlighting